packages/cli-lib/api/appPipeline.js

```diff
@@ -3,18 +3,18 @@ const APP_PIPELINE_API_PATH = '/app-pipeline/v1';
 
 async function deployApp(accountId, appFolderPath) {
   return http.post(accountId, {
-    uri: `${APP_PIPELINE_API_PATH}/deploy/async`,
-    body: {
+    url: `${APP_PIPELINE_API_PATH}/deploy/async`,
+    data: {
```

Comment:
Here is an example of different expected property names between `request` and `axios` that will need to be updated throughout the code.